aardvark/view.py

```diff
@@ -22,6 +24,31 @@ def __init__(self):
         super(RoleSearch, self).__init__()
         self.reqparse = reqparse.RequestParser()
 
+    def combine(self, aa):
+        del aa['count']
+        del aa['page']
+        del aa['total']
+
+        usage = dict()
+        for arn, services in aa.items():
+            for service in services:
+                namespace = service.get('serviceNamespace')
+                last_authenticated = service.get('lastAuthenticated')
+                if namespace not in usage:
+                    usage[namespace] = service
+                else:
+                    if last_authenticated > usage[namespace]['lastAuthenticated']:
+                        usage[namespace] = service
+                        del usage[namespace]['totalAuthenticatedEntities']
+
+        for namespace, service in usage.items():
+            last_authenticated = service['lastAuthenticated']
+            dt_last_authenticated = datetime.datetime.fromtimestamp(last_authenticated / 1e3)
+            dt_starting = datetime.datetime.utcnow() - datetime.timedelta(days=90)
+            usage[namespace]['USED_LAST_90_DAYS'] = dt_last_authenticated > dt_starting
```

Comment:
Added `USED_LAST_90_DAYS` as a bool so parsing the epoch `lastAuthenticated` isn't always required.